docs/requirement-sets/powerpoint/powerpoint-preview-apis.md

```diff
@@ -16,8 +16,7 @@ The first table provides a concise summary of the APIs, while the subsequent tab
 
 | Feature area | Description | Relevant objects |
 |:--- |:--- |:--- |
-| Slide management | Adds support for adding slides as well as managing slide layouts and slide masters. | [Slide](/javascript/api/powerpoint/powerpoint.slide)<br>[SlideLayout](/javascript/api/powerpoint/powerpoint.slidelayout)<br>[SlideMaster](/javascript/api/powerpoint/powerpoint.slidemaster)|
-| Shapes | Adds support for getting references to the shapes in a slide. | [Shape](/javascript/api/powerpoint/powerpoint.shape) |
+| Selection APIs | Adds more ways to programmatically select slides, shapes, and text ranges. | See the table later in this article. |
 
 ## API list
 
```

Comment:
There's a typo in the next sentence (on line 23) that mentions Excel. It should say PowerPoint